homework/test/test-list.js

```diff
@@ -2,7 +2,11 @@ describe('Test > ', function() {
   beforeEach(function() {
   });
 
-  it('will test some pure functions', function() {
+  it('my pure testfunctions', function() {
     // Write any pure function assertion here.
+    var a = 1;
+    var b = 2;
+    var sum = a+b;
+    assert.equal(3,sum);
```

Comment:
**WRONG**: you need to **define a pure function (method) in one manager, and then call it here.**